policies/PrivacyPolicy.html

```diff
@@ -50,9 +50,9 @@
   <div><br /></div>
   <div><strong>What personal information do we process?</strong> When you visit, use, or navigate our Services, we may process personal information depending on how you interact with us and the Services, the choices you make, and the products and features you use. Learn more about <a href="#personalinfo">personal information you disclose to us</a>.</div>
   <div><br /></div>
-  <div><strong>Do we process any sensitive personal information?</strong> We do not process sensitive personal information.</div>
+  <div><strong>Do we process any sensitive personal information?</strong> We collect and process the user's date of birth for reasons of ensuring users meet our minimum age threshold. Learn more about <a href="#personalinfo">sensitive information you disclose to us</a>.</div>
```

Comment:
Is DOB classed as sensitive? When we generated this policy we included DOB in the forms and it did not get filled out here.

![image](https://github.com/user-attachments/assets/842fb55f-d7e7-41d6-8827-88d9c89a0454)